src/index.html

```diff
@@ -4,10 +4,265 @@
+              <section class="nav__button">
+                <a href="#home">
+                  <img
+                    src="/images/Close.svg"
+                    alt="Close-button"
+                    class="nav__button-close"
+                    data="close-menu"
+                  >
+                </a>
+              </section>
+            </article>
+            <nav class="nav__catalog">
+              <a href="#home" class="nav__link">HOME</a>
+              <a href="#recommended" class="nav__link">RECOMMENDED</a>
+              <a href="#categories" class="nav__link">CATEGORIES</a>
+              <a href="#buy" class="nav__link">HOW TO BUY</a>
+              <a href="#contacts" class="nav__link">CONTACTS</a>
+              <section class="nav__contacts">
+                <a href="tel:+1 234 5555-55-55" class="nav__contact-number">+1 234 5555-55-55</a>
+                <a href="tel:+1 234 5555-55-55" class="nav__contact-order">call to order</a>
+              </section>
+            </nav>
+          </article>
+        </nav>
+
+        <article class="header__advertising">
+          <span class="header__vector"></span>
+          <p class="header__advice">The world shades.<br>Your music shines.</p>
+        </article>
+      </article>
+    </header>
+
+    <main class="main">
+      <p class="main__recom-title">Recommended</p>
+
+      <article
+        class="main__recommended"
+        id="recommended"
+        data-aos="fade-up"
+        data-aos-duration="1200"
+      >
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Smart-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">Bose portable Smart speaker</p>
+          <p class="card__category">Smart home</p>
+          <p class="card__price">$ 399.00</p>
+        </section>
+
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Bluetooth-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">SoundLink Flex Bluetooth speaker</p>
+          <p class="card__category">Portable bluetooth</p>
+          <p class="card__price">$ 149.00</p>
+        </section>
+
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Color-bluetooth-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">
+            SoundLink Color Bluetooth speaker II
+          </p>
+          <p class="card__category">Portable bluetooth</p>
+          <p class="card__price">$ 129.00</p>
+        </section>
+      </article>
+      <article class="main__category-box" id="categories">
```

Comment:
<img width="1020" alt="Screenshot 2023-02-14 at 4 14 31 PM" src="https://user-images.githubusercontent.com/35467335/218763889-0afa9687-a659-463c-b717-261729eed038.png">

You have too big distance between your title and photos, check your margins.